tailcfg/tailcfg.go

```diff
@@ -1749,15 +1750,6 @@ type ControlIPCandidate struct {
 //
 // TODO(bradfitz): start migrating the imperative ones to c2n requests.
 type Debug struct {
-	// LogHeapPprof controls whether the client should log
-	// its heap pprof data. Each true value sent from the server
-	// means that client should do one more log.
-	LogHeapPprof bool `json:",omitempty"`
```

Comment:
Kinda annoying to use these fields for old clients if we don't have them in tailcfg. I guess we could do tailcfgout. Also debug, so don't really care if we lose the ability for old clients

Reply:
We don't use these old fields. This was here for debugging iOS memory usage. I think I was the only one to ever use it, on my own phone only.

Because there was no other way to get the pprof from the phone.